docs/apache-airflow-providers-slack/notifications/slack_notifier_howto_guide.rst

```diff
@@ -20,7 +20,7 @@ How-to Guide for Slack notifications
 
```

Comment:
I cannot add a comment before "How-to Guide for Slack notifications" but that's where I'd put it
```suggestion

.. _howto/notifier:SlackNotifier:

```